themes/jb/layouts/_default/single.html

```diff
@@ -27,7 +27,7 @@ <h1 class="title">{{.Title}}</h1>
       {{ range $key, $value := . }}
       <!-- currently 90% of images grabbed from: https://linuxunplugged.com/subscribe -->
         <div class="p-1 is-flex subscribe-logos">
-          <a class="is-flex is-align-content-center is-justify-content-center" href="{{ $value }}">
+          <a class="is-flex is-align-content-center is-justify-content-center" href="{{ $value }}" target="_blank" rel="noreferrer">
```

Comment:
Is there a reason why you're adding the `rel="noreferrer"`? While _technically_ it's a bit more privacy conscious I'd almost say that it's a good thing to have us referring people to other websites (it shows JB.com is driving traffic to their website).

```suggestion
          <a class="is-flex is-align-content-center is-justify-content-center" href="{{ $value }}" target="_blank" >
```

Reply:
opens you up to a vulnerability: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/ I can switch it to rel="noopener" which will still allow referal information.

Reply:
So, what's the difference between that and someone just putting `window.location = "https://google.com"` with a script tag on the page?

Reply:
The way this vulnerability works is if the site you link the person too has been compromised, they can tell the browser to redirect the previous tab aka the JB site to to any site they want in the background. Since we have no control over other podcasts hosting/directory sites it should be taken as a precautionary action to prevent this type of attack. This youtuber demonstrates how you can exploit this: https://youtu.be/wWzlN096DvA

Reply:
Interesting...while technically it's up to the user to still be aware of their URL address changing (because it changes the location of the previously opened window and doesn't just modify it in place or something), it doesn't hurt to have it I guess.

If you're trying to handle that though, then you're missing an argument of `noopener` as well. So, something like this:

```suggestion
          <a class="is-flex is-align-content-center is-justify-content-center" href="{{ $value }}" target="_blank" rel="noopener noreferrer">
```

at least according to the article you linked to: https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/

Reply:
BTW, the YT video you linked to even says that chrome defaults to noopener when not specified, but again doesn't hurt to be a bit more redundant: https://youtu.be/wWzlN096DvA?t=248